guides/getting_started/comparison_to_alternatives.md

```diff
@@ -0,0 +1,101 @@
+
+Today, Lucene and Elasticsearch are dominant players in the open source search engine landscape. They both are solid solutions for a lot of different use cases in search, and also for building your own recommendation engine. They are good general products, but they require to be configured properly to get similar results to those of MeiliSearch or Algolia.  
+
+
+### Closed Source
+
+#### Algolia
+
+Algolia is a company providing a search engine on a SaaS model. Its software is closed source. In its early stages, Algolia offered mobile search engines that could be embedded in apps, facing the challenge of implementing the search algorithms from scratch. From the very beginning, the decision was made to build a search engine directly dedicated to the end users, i.e. implementing search within mobile apps or websites.  
+Algolia successfully demonstrated over the past few years how critical tolerating typos is in order to improve the users’ experience, and in the same way, its impact on reducing bounce rate and increasing conversion.  
+
+Apart from Algolia, a wide range of SaaS products exist on the Search Engine Market. Most of them use Elasticsearch, and fine tune its settings in order to have a custom and personalized solution.  
+
+#### Swiftype
+
+Swiftype is a search service provider specialized in website search and analytics. Swiftype was founded in 2012 by Matt Riley and Quin Hoxie, and is now owned by Elastic since November 2017. It is a end-to-end solution built on top of Elasticsearch, meaning it has the ability to leverage the Elastic Stack.  
+
+#### Doofinder
+
+Doofinder is a paid on-site search service which was developed to integrate to any website with very little configuration. Doofinder is used by online stores to increase their sales, aiming to facilitate the purchase process.  
+
+## Conclusions
+
+MeiliSearch offers the same features as Algolia but is intended to be a more simplified alternative. In addition, providing an open source code allows to unleash a large number of opportunities for monetization: SaaS, support services, enterprise-features, and so on.  
+Today, Algolia is no longer perceived as a dev tool, a developer friendly tool, since Algolia's marketing strategy is now mainly focused on large businesses; especially on e-commerce market where instant search is proving effective in bringing a strong added value. Since MeiliSearch is open source, it will be integrated in relatively smaller projects and will be managed by the development team rather than the marketing team or the product team. MeiliSearch will continue growing driven by its projects! Our ambition for MeiliSearch is to become the reference technology in the search engine market.  
```

Comment:
```suggestion
Today, Algolia is no longer perceived as a developer friendly tool, since Algolia's marketing strategy is now mainly focused on large businesses; especially on e-commerce market where instant search is proving effective in bringing a strong added value. Since MeiliSearch is open source, it will be integrated in relatively smaller projects and will be managed by the development team rather than the marketing team or the product team. MeiliSearch will continue growing driven by its projects! Our ambition for MeiliSearch is to become the reference technology in the search engine market.  
```